contributors/contributors/community.md

```diff
@@ -103,45 +86,45 @@ All contributors Level 2 and above are eligible for a [custom-printed Mattermost
 
 Community Coordinator: Jason Blais and Ben Schumacher
 
-#### [Translation Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Translation Contributors
 
 Translation Contributors can create and update translations to existing or new Mattermost languages at [https://translate.mattermost.com](https://translate.mattermost.com/). In addition, community members can become “Language owners” and help review, coordinate and supervise the translation of Mattermost into languages they speak.
 
 Community Coordinator: Jason Blais
 
-#### [Documentation Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Documentation Contributors
 
 Documentation Contributors update, edit, and create new documentation, help text, training, and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).
 
 Community Coordinator: Jason Blais and Justine Geffen
 
-#### [Quality Assurance Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Quality Assurance Contributors
 
 Quality Assurance Contributors help find, confirm and report product defects via our process at [https://www.mattermost.org/filing-issues/](https://www.mattermost.org/filing-issues/). Anyone who encounters a verifiable defect is welcome to file a bug on the system.
 
 Bonus: When QA Contributor finds and reports a bug on a release candidate build of Mattermost prior to the final release of the system, the can be awarded a [Mattermost Bug Hunter Award](https://forum.mattermost.org/t/mattermost-bug-hunter-awards/4979) for their contribution.
 
 Community Coordinator: TBD
 
-#### [Knowledge Base Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Knowledge Base Contributors
 
 Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and the [Mattermost community server](https://community.mattermost.com/).
 
 Community Coordinator: Amy Blais
 
-#### [Deployment Solution Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Deployment Solution Contributors
 
 Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
```

Comment:
```suggestion
Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build in line with Mattermost quality standards.
```

Reply:
Wondering if a link to the SA channel or other relevant channel would be useful? Might also be a good jumping off point for communication between contributor and Mattermost.

Reply:
Good idea! Is the SA channel public?

Reply:
Ah, good point. It's not. :(